src/App/screens/Instructor/components/Icon/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+export const sizes = {
+  1: '',
+  2: 'fa-lg',
+  3: 'fa-2x',
+  4: 'fa-3x',
+  5: 'fa-4x',
+  6: 'fa-5x',
+}
+
+export const types = {
+  'menu': 'bars',
+  'step-incomplete': 'square-o',
+  'step-complete': 'check-square-o',
+  'close': 'close',
+  'more-info': 'info-circle',
+}
```

Comment:
Mapping project prototypes to library values; we did this at my last job and it was very useful for dynamically creating a living style guide, validating proptypes, and swapping out libraries etc.